launch.go

```diff
@@ -268,6 +268,11 @@ func createEnvironment(base map[string]string, secrets screwdriver.Secrets) []st
 	for k, v := range combined {
 		envStrings = append(envStrings, strings.Join([]string{k, v}, "="))
 	}
+
+	for k, v := range build.Environment {
```

Comment:
@stjohnjohnson We can discuss about that more. But this PR doesn't change the behaviors. It has always been adding ENV this way inside the launcher. This is to refactor/clean up code.